### ISSUE TITLE
No login into Artemis console if external auth service without caCertSecretName is used

### ISSUE THREAD

**Issue by calohmn** [EnMasseProject/enmasse#1880]
**Describe the bug**
If an address space with an external auth service is used - where caCertSecretName has been omitted, the login into the Artemis console doesn't work.

**To Reproduce**
1. Create an address space with a spec like this
````
  "spec": {
    "type": "standard",
    "plan": "unlimited-standard-without-mqtt",
    "authenticationService": {
      "type": "external",
      "details": {
        "host": "[hostname]",
        "port": 5671,
        "saslInitHost": "default",
        "clientCertSecretName": ""
      }
    }
  }
````
Note that "caCertSecretName" is not set - requires the authservice to use a proper cert (not a self-signed one).

2. Create a port-forwarding for the broker:
````
kubectl port-forward -n hono broker-pooled-default-0 9999:8161
````
3. Try to access Artemis console via `https://localhost:9999`
-> Credentials are not accepted.

**Expected behavior**
Login to Artemis console works.

**Additional context**
Log output of Artemis container:
````
2018-10-29 09:44:20,754 WARN  [io.enmasse.artemis.sasl_delegation.SaslDelegatingLogin] javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
2018-10-29 09:44:20,755 WARN  [io.hawt.system.Authenticator] Login failed due Exception attempting to authenticate using SASL delegation
````

**Comment by calohmn** [comment]
This relates to #1580 and the fix for that.
Applies to `SaslDelegatingLogin.createSSLContext()`.

**Comment by calohmn** [comment]
A solution would be to replace the keytool invocation in `launch.sh` with this:
````
    local num_certs=$(grep 'END CERTIFICATE' /etc/authservice-ca/tls.crt | wc -l)
    if [ "${num_certs}" == "1" ]; then
        keytool -import -noprompt -file /etc/authservice-ca/tls.crt -alias firstCA -deststorepass ${TRUSTSTORE_PASS} -keystore ${AUTH_TRUSTSTORE_PATH}
    else
        # import certs individually
        for N in $(seq 0 $((${num_certs} - 1))); do
          cat /etc/authservice-ca/tls.crt |
            awk "n==$N { print }; /END CERTIFICATE/ { n++ }" |
              keytool -import -noprompt -alias CA-${N} -deststorepass ${TRUSTSTORE_PASS} -keystore ${AUTH_TRUSTSTORE_PATH}
        done
    fi
````
However, for the 133 certs in the actual scenario, this takes quite long - 28 seconds.

I wonder whether it wouldn't be better to somehow use the already existing standard java cacerts file.

**Comment by lulf** [comment]
I saw that approach on SO :/ agreed copying the system trust store and adding to that is better

**Comment by lulf** [comment]
Fixed by #1881
